zilliqa/src/state.rs

```diff
@@ -205,9 +214,15 @@ impl State {
     }
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize)]
+#[derive(Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize)]
 pub struct Address(pub H160);
 
+impl fmt::Debug for Address {
```

Comment:
I like the `Display` fix, but out of curiosity what's the reason the automatic derive was not sufficient for `Debug`?

Reply:
The default implementation of it prints H160 as "0x8ff....0ac" - which is annoying imo